manifests/backports.pp

```diff
@@ -45,6 +45,7 @@
     } else {
       $distid = 'ubuntu'
       $release_real = $::lsbdistcodename ? {
+        'rebecca'  => 'trusty',
```

Comment:
I know this makes the change bigger, but can you align the arrows in the selector?